extras/java8-tests/src/test/java/org/apache/spark/streaming/Java8APISuite.java

```diff
@@ -440,8 +441,13 @@ public void testPairFlatMap() {
   public static <T extends Comparable<T>> void assertOrderInvariantEquals(
     List<List<T>> expected, List<List<T>> actual) {
     expected.forEach((List<T> list) -> Collections.sort(list));
-    actual.forEach((List<T> list) -> Collections.sort(list));
-    Assert.assertEquals(expected, actual);
+    ArrayList<ArrayList<T>> sortedActual = new ArrayList<ArrayList<T>>();
```

Comment:
I think this can be `List<List<T>> sortedActual = new ArrayList<>();` right? And similarly below.